service/Dockerfile

```diff
@@ -2,10 +2,12 @@
 FROM openjdk:8-jre
 
 WORKDIR /usr/local/src
-COPY . /usr/local/src
+COPY . /usr/local/src/
```

Comment:
Ah, I missed that. I still feel as though it is a bit confusing (I think it is unlikely that people will remember to dig into the `.dockerignore` file to alter copy behavior).

What is the value in copying things `src`, `target`, and `setup.py`? It seems like then we copy the JAR in twice, but then delete one of the copies.